### PR TITLE
The grand emotening part 1

### PR DESCRIPTION
**Pull request by Gboster-0**
## About The Pull Request

Updates our emotes to recent-ish TG

The following lists with all the PR's are below, keep in mind the PR's can repeat across multiple lists (its easier for me to organize that way)

<details>
<summary>Ported PR's changing the file "code/modules/mob/living/emote.dm"</summary>

- https://github.com/tgstation/tgstation/pull/69127 
- https://github.com/tgstation/tgstation/pull/69007
- https://github.com/tgstation/tgstation/pull/68948
- https://github.com/tgstation/tgstation/pull/68870 
- https://github.com/tgstation/tgstation/pull/66321
- https://github.com/tgstation/tgstation/pull/65169
- https://github.com/tgstation/tgstation/pull/64490
- https://github.com/tgstation/tgstation/pull/62985
- https://github.com/tgstation/tgstation/pull/62639
- https://github.com/tgstation/tgstation/pull/62515
- https://github.com/tgstation/tgstation/pull/62394
- https://github.com/tgstation/tgstation/pull/59981
- https://github.com/tgstation/tgstation/pull/59797
- https://github.com/tgstation/tgstation/pull/59680
- https://github.com/tgstation/tgstation/pull/59645
- https://github.com/tgstation/tgstation/pull/59585 
- https://github.com/tgstation/tgstation/pull/59245
- https://github.com/tgstation/tgstation/pull/59479
- https://github.com/tgstation/tgstation/pull/59495
- https://github.com/tgstation/tgstation/pull/56732
- https://github.com/tgstation/tgstation/pull/56698
- 
</details>

<details>
<summary>Ported PR's changing the file "code/datums/emotes.dm"</summary>

- https://github.com/tgstation/tgstation/pull/62226
- https://github.com/tgstation/tgstation/pull/62206
- https://github.com/tgstation/tgstation/pull/60772
- https://github.com/tgstation/tgstation/pull/60432
- https://github.com/tgstation/tgstation/pull/60266
- https://github.com/tgstation/tgstation/pull/59645
- https://github.com/tgstation/tgstation/pull/59245
- https://github.com/tgstation/tgstation/pull/59479
- https://github.com/tgstation/tgstation/pull/59495
- https://github.com/tgstation/tgstation/pull/58821
- https://github.com/tgstation/tgstation/pull/57114

</details>

<details>
<summary>Ported PR's changing the file "code/game/objects/items/hand_items.dm"</summary>

- https://github.com/tgstation/tgstation/pull/65169
- https://github.com/tgstation/tgstation/pull/64512
- https://github.com/tgstation/tgstation/pull/63916
- https://github.com/tgstation/tgstation/pull/62594
- https://github.com/tgstation/tgstation/pull/59109
- https://github.com/tgstation/tgstation/pull/63510
- https://github.com/tgstation/tgstation/pull/62594
- https://github.com/tgstation/tgstation/pull/61123
- https://github.com/tgstation/tgstation/pull/61623
- https://github.com/tgstation/tgstation/pull/61590
- https://github.com/tgstation/tgstation/pull/61375
- https://github.com/tgstation/tgstation/pull/61392
- https://github.com/tgstation/tgstation/pull/60302
- https://github.com/tgstation/tgstation/pull/61183
- https://github.com/tgstation/tgstation/pull/59645
- https://github.com/tgstation/tgstation/pull/59260
- https://github.com/tgstation/tgstation/pull/57304
- https://github.com/tgstation/tgstation/pull/57797
- https://github.com/tgstation/tgstation/pull/58137
- https://github.com/tgstation/tgstation/pull/57269
- https://github.com/tgstation/tgstation/pull/56912
- https://github.com/tgstation/tgstation/pull/56732
- https://github.com/tgstation/tgstation/pull/56698

</details>

- Also this PR modularized the fixer badge to be a subtype of the lawyer's one, instead of overriding it (emotes conflicted)
- And changes `deathmessage` to `death_message` since simplemob TG emote stuff changes

Things to fix:

~- [ ] Custom emotes showing an NTOS error when cancelled~ TGUI issue, best to be handled in another PR
~- [ ] Slaps dealing 1 sanity damage~ will come in emotening part 2

## Why It's Good For The Game

This gives a LOT of new roleplay possibilities and makes the whole system more refined, whats not to love

## Changelog
:cl:
add: Improved emotes, a lot, a lotta lot
code: emote related variables in abnormality files were made more readable
code: fixer's badge is now its own subtype instead of overriding the lawyers badge
/:cl:

### CODE REVIEW

File: code/__DEFINES/achievements.dm [hunk 1]
```diff
@@ -4,33 +4,35 @@
...
+#define MEDAL_GOTTEM "GOTTEM"
+#define MEDAL_ASCENSION "Ascension"
+#define MEDAL_FRENCHING "FrenchingTheBubble"
+#define MEDAL_ASH_ASCENSION "Ash"
+#define MEDAL_FLESH_ASCENSION "Flesh"
+#define MEDAL_RUST_ASCENSION "Rust"
+#define MEDAL_VOID_ASCENSION "Void"
+#define MEDAL_TOOLBOX_SOUL "Toolsoul"
+//#define MEDAL_CHEM_TUT "Beginner Chemist"
+#define MEDAL_HOT_DAMN "Hot Damn!"
```
Comment (LanceSmites328) [inline code/__DEFINES/achievements.dm:35]: Would like these to get their unified indentation back (Where they're all on the same column)

File: code/__DEFINES/colors.dm [hunk 2]
```diff
@@ -50,14 +50,15 @@
...
-#define COLOR_STRONG_VIOLET				"#6927c5"
+#define COLOR_PINK "#FFC0CB"
+#define COLOR_LIGHT_PINK "#ff3cc8"
+#define COLOR_MOSTLY_PURE_PINK "#E4005B"
+#define COLOR_BLUSH_PINK "#DE5D83"
+#define COLOR_MAGENTA "#FF00FF"
+#define COLOR_STRONG_MAGENTA "#B800B8"
+#define COLOR_PURPLE "#800080"
+#define COLOR_VIOLET "#B900F7"
+#define COLOR_STRONG_VIOLET "#6927c5"
```
Comment (LanceSmites328) [inline code/__DEFINES/colors.dm:61]: Would like these to get their unified indentation back (Where they're all on the same column)

File: code/game/objects/items/hand_items.dm [hunk 3]
```diff
@@ -0,0 +1,605 @@
...
+	user.visible_message(span_danger("[user] begins giving [target] a [message_others]!"), span_warning("You start giving [target] a [message_others]!"), vision_distance=COMBAT_MESSAGE_RANGE, ignored_mobs=target)
+	to_chat(target, span_userdanger("[user] starts giving you a [message_target]!"))
+
+	if(!do_after(user, 1.5 SECONDS, target))
+		to_chat(user, span_warning("You fail to give [target] a noogie!"))
+		to_chat(target, span_danger("[user] fails to give you a noogie!"))
+		return
+
+	if(brutal_noogie)
+		to_chat(user, span_warning("You are not supposed to see this!")) // LOBOTOMYCORPORATION ADDITION - i needed some code here so it wouldn't give warnings, enjoy dis
```
Comment (LanceSmites328) [inline code/game/objects/items/hand_items.dm:164]: brutal_noogie variable is local, just... don't add it?

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

**Review by LanceSmites328 — DISMISSED** [review]

File: code/game/objects/items/hand_items.dm (hunk 3 above)
Comment (Gboster-0) [reply code/game/objects/items/hand_items.dm:164]: for future reviewers: Keeping the brutal_noogie variable for less possible future conflicts with cherry-picking from TG. Conversation on discord

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**